core-plugins/src/main/java/co/cask/hydrator/plugin/batch/joiner/Joiner.java

```diff
@@ -52,6 +58,11 @@
   "records from non-required inputs will only be present if they match join criteria. If there are no required " +
   "inputs, outer join will be performed")
 public class Joiner extends BatchJoiner<StructuredRecord, StructuredRecord, StructuredRecord> {
+
+  public static final String JOIN_OPERATION_DESCRIPTIOPN = "Join Operation";
+  public static final String IDENTITY_OPERATION_DESCRIPTION = "Identity output as a part of JOIN";
+  public static final String RENAME_OPERATION_DESCRIPTION = "Renaming field as a part of JOIN";
```

Comment:
It is redundant to use 'field' in these descriptions, as all these operations are always on fields. 'Renamed as part of a join'.

Reply:
fixed.